bootstrap_cfn/salt_utils.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+import sys
+sys.path.append('/usr/local/bootstrap-cfn')
```

Comment:
Good shout, actually needs ``sys.path.append(os.path.dirname(__file__) + '/..')`` I think the folder of the module is always in sys.path.